templates/testpmd/scripts/change-nic-driver.sh

```diff
@@ -11,6 +11,14 @@ if [ "$driver" == "vfio-pci" ]
 then
   echo "Driver is vfio-pci, no extra action needed."
   exit
+elif [[ "$driver" == *"mlx"* ]]
+then
+  echo "Driver is a Mellanox driver, please refer to https://doc.dpdk.org/guides/nics/mlx5.html#usage-example in order to replace it."
+  exit
+elif [ "$driver" != "iavf" ]
+then
+  echo "Driver is $driver, and cannot be handled here."
+  exit
```

Comment:
Why the script exist if the driver is not `iavf`? I think it should change it either way.